testing_config.py

```diff
@@ -39,13 +39,16 @@
 from google.appengine.ext import vendor
 vendor.add(lib_path) # add third party libs to "lib" folder.
 
-from google.appengine.ext import ndb
+# from google.appengine.ext import ndb
+from google.cloud import ndb
 from google.appengine.ext import testbed
 
 os.environ['DJANGO_SECRET'] = 'test secret'
 os.environ['SERVER_SOFTWARE'] = 'test ' + os.environ.get('SERVER_SOFTWARE', '')
 os.environ['CURRENT_VERSION_ID'] = 'test.123'
-
+os.environ['DJANGO_SETTINGS_MODULE'] = 'settings'
+os.environ['DATASTORE_EMULATOR_HOST'] = 'localhost:15606'
+ 
```

Comment:
These two environment variables are needed to be added while running tests.

Reply:
Makes sense.